packages/mini-browser/src/browser/mini-browser-content.ts

```diff
@@ -590,6 +657,8 @@ export class MiniBrowserContent extends BaseWidget {
                 if (this.getToolbarProps() === 'read-only') {
                     this.input.title = `Open ${url} In A New Window`;
                 }
+                clearTimeout(this.frameLoadTimeout);
+                this.frameLoadTimeout = setTimeout(this.onFrameTimeout.bind(this), 3000) as unknown as number;
```

Comment:
`this.frameLoadTimeout = window.setTimeout(this.onFrameTimeout.bind(this), 3000);`?

Reply:
Thanks, I will try this (I was quite confused by why TypeScript made me do all these type conversions).